websites/F/Facebook/dist/metadata.json

```diff
@@ -22,8 +22,8 @@
   },
   "url": "www.facebook.com",
   "version": "3.2.2",
-  "logo": "https://i.imgur.com/jWYZLvY.png",
-  "thumbnail": "https://i.imgur.com/WcKmon0.png",
+  "logo": "https://i.imgur.com/nS9sZn6.png",
```

Comment:
I strongly disagree with this change. The logo isn't circular nor round, but if you have a strictly round logo with white borders it will look very bad